bin/improver-generate-topography-bands-mask

```diff
@@ -53,10 +53,10 @@ THRESHOLDS_DICT = {'bounds': [[-500., 50.], [50., 100.], [100., 150.],
 def main():
     """Load in arguments and get going."""
     parser = ArgParser(
-        description=('Read input orography and landmask fields. Return a '
-                     'a cube of masks, where each mask excludes data below '
-                     'or equal to the lower threshold, and excludes data above'
-                     ' the upper threshold.'))
+        description=('Reads input orography and landmask fields. Creates a '
+                     'set of masks, where each mask excludes data below or '
```

Comment:
Since "set" is also a Python data type - an external collaborator might missinterpret this.
I suggest using the word "series" since this has the same kind of meaning but no conflict with types.

Reply:
"collection" is another possibility